### PR TITLE
Avoid duplicate definitions of `gtk_fixed` operations.

### PR DESCRIPTION
**Pull request by samth**

### CODE REVIEW

File: gui-lib/mred/private/wx/gtk/group-panel.rkt [hunk 1]
```diff
@@ -6,6 +6,7 @@
          "window.rkt"
          "client-window.rkt"
          "panel.rkt"
+         "frame.rkt"
```
Comment (mflatt) [inline gui-lib/mred/private/wx/gtk/group-panel.rkt:9]: Doesn't seem like a big deal, but is this import needed?

File: gui-lib/mred/private/wx/gtk/tab-panel.rkt [hunk 2]
```diff
@@ -9,6 +9,7 @@
          "types.rkt"
          "widget.rkt"
          "message.rkt"
+         "frame.rkt"
```
Comment (mflatt) [inline gui-lib/mred/private/wx/gtk/tab-panel.rkt:12]: This import doesn't seem needed, either.

**Review by mflatt — APPROVED** [review]

File: gui-lib/mred/private/wx/gtk/group-panel.rkt (hunk 1 above)
Comment (samth) [reply gui-lib/mred/private/wx/gtk/group-panel.rkt:9]: It's needed because they were getting the functions from the other definition before.

Comment (mflatt) [reply gui-lib/mred/private/wx/gtk/group-panel.rkt:9]: Oh, I see. Ok!

**Pull request merged**